### PR TITLE
[docs]: vector store integration pages

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/integrations/vectorstores/faiss_index/index.faiss
Comment (efriis) [inline docs/docs/integrations/vectorstores/faiss_index/index.faiss:1]: do we want to commit this or remove?

**Review by efriis — APPROVED** [review]
small nit otherwise lgtm

File: docs/docs/integrations/vectorstores/faiss_index/index.faiss
Comment (isahers1) [reply docs/docs/integrations/vectorstores/faiss_index/index.faiss:1]: remove I think, it was leftover from a previous PR. unless I'm missing something it's just a file generated when you run the notebook

Comment (efriis) [reply docs/docs/integrations/vectorstores/faiss_index/index.faiss:1]: nice added a .gitignore so it doesn't get committed back

**Pull request merged**